README.md

```diff
@@ -33,6 +33,42 @@ this repository.
 * setup.cfg 
 * variables.json.example - contains data needed by the tests.
 
+# Updating Python Requirements
+
+If you want to just update the python libraries on your box, then become
+root, go to the root of this source tree, and run the command:
+
+    pip install -r requirements.txt
+
+If however you want use a python virtual environment (see 
+https://virtualenv.pypa.io/en/stable/) then you can use this set 
+of commands:
+
+    virtualenv ${workspace}.venv
+    . ${workspace}.venv/bin/activate
+    pip install -r requirements.txt
+
+Note, git is configured to ignore files that end in `.venv` and
```

Comment:
git isn't configured to ignore them by default. If a developer is using the '.venv' extension for their virtual environments, it would be better to add it to their own workstation global .gitignore file